meta.yml

```diff
@@ -16,6 +16,6 @@ data:
   analytics_url: "https://analytics.usa.gov/data/live/second-level-domains.csv"
 
   # DAP URL where we source public .gov subdomains from.
-  analytics_subdomains_url: "https://analytics.usa.gov/data/live/sites.csv"
+  analytics_subdomains_url: "https://analytics.usa.gov/data/live/sites-extended.csv"
```

Comment:
We'll need to also update the deploy script in analytics-reporter to generate and publish this report.